.githooks/pre-commit

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+black . --check
+flake8
```

Comment:
these should both operate on `xbstrap/` + `setup.py` as they otherwise risk messing up files in ignored subdirs, they also should only operate on indexed data (`black --stdin-filename` and `flake8 --stdin-display-name`) as to not mess up noncommited changes

also `isort` (+ `isort --filename`) should be included